test/basic_get_save_v1_test.py

```diff
@@ -181,8 +330,6 @@ def test_get_all(self):
         self.assertEqual(len(data),1)
         self.assertEqual(data[0]['data']['scientific_name'],'Rhodobacter CACIA 14H1')
         self.assertTrue('features' in data[0]['data'])
-        self.assertTrue('cdss' in data[0]['data'])
```

Comment:
why you want to remove those?

Reply:
The genome that was being tested added in blank 'cdss' and 'mrnas' arrays making it an "intermediate" genome. My downgrade function consolidates all types into the features array like the old genome did because that's what most apps will expect. I took these checks out because I an not going to add the arrays in the new version, which should make it invariant when passed though the downgrade function.